pyproject.toml

```diff
@@ -33,7 +33,7 @@ python = "~3.11.9"
 boto3 = "1.26.86"
 botocore = "1.29.86"
 debugpy = "1.6.7"
-Django = "3.2.19"
+Django = "5.1.1"
 django-allauth = "0.54.0"
 django-cors-headers = "4.0.0"
 django-csp = "3.7"
```

Comment:
Would be worth checking notes for all of these other Django-related packages to see if they support Django 5.x.

We're probably best off bumping all of them up to vLatest tbh

Reply:
have done most of these, though to note:
- the update for `django-allauth` seems to be breaking, would need some more research into how we interact with this library
- `django-use-email-as-username` is now [no longer actively maintained](https://github.com/jmfederico/django-use-email-as-username)

Reply:
agreed in catchup to ensure `django-allauth` is compatible with django 5.0. backlog ticket to be raised for `django-use-email-as-username`

Reply:
have updated `django-allauth` to a version known to work with django 5.0. won't go any further as there are many breaking changes to consider https://docs.allauth.org/en/latest/release-notes/recent.html

Reply:
have added [PC-1634](https://beisdigital.atlassian.net/browse/PC-1634), [PC-1635](https://beisdigital.atlassian.net/browse/PC-1635) & [PC-1636](https://beisdigital.atlassian.net/browse/PC-1636) to cover removing use email as username & implementing a package auditor across our services